test/performance/config/performance-reporter.ts

```diff
@@ -118,7 +118,7 @@ export function curateResults(
 	return Object.fromEntries(
 		Object.entries( output )
 			// Reduce the output to contain taken metrics only.
-			.filter( ( [ _, value ] ) => value !== undefined )
+			.filter( ( item ) => item[ 1 ] !== undefined )
```

Comment:
Let's use a more minimal change here, we can remove the unused `_` binding:

```suggestion
			.filter( ( [ , value ] ) => value !== undefined )
```